site/templates/application/application.jet

```diff
@@ -76,6 +78,7 @@
     <s72-cookie-consent></s72-cookie-consent>
 
     {{yield googleTagManagerNoScript()}}
+    {{yield pixelNoScript()}}
```

Comment:
Do we need to yield this twice?

Reply:
thats the no script block. i separated it and chucked it in next to the google no script block. it will only load if theres no cookie consent feature enabled

Reply:
Wow... am I actually that blind!?!?